### ISSUE TITLE
Periodic kernel implementation seems incorrect

### ISSUE THREAD

**Issue by vr308** [pymc-devs/pymc#3540]
The following snippet is from the code base, in the last line it should be multiplied by -2 and not -0.5 (as is with other kernels). In fact, the equation in the comment is correct. 
 
## Description of your problem

```class Periodic(Stationary):
    r"""
    The Periodic kernel.
    .. math::
       k(x, x') = \mathrm{exp}\left( -\frac{2 \mathrm{sin}^2(\pi |x-x'| \frac{1}{T})}{\ell^2} \right)
    """

    def __init__(self, input_dim, period, ls=None, ls_inv=None, active_dims=None):
        super().__init__(input_dim, ls, ls_inv, active_dims)
        self.period = period

    def full(self, X, Xs=None):
        X, Xs = self._slice(X, Xs)
        if Xs is None:
            Xs = X
        f1 = X.dimshuffle(0, "x", 1)
        f2 = Xs.dimshuffle("x", 0, 1)
        r = np.pi * (f1 - f2) / self.period
        r = tt.sum(tt.square(tt.sin(r) / self.ls), 2)
        return tt.exp(-0.5 * r)
```

**Comment by fonnesbeck** [comment]
Actually, this is the same parameterization as those in GPy/GPFlow. So its the docstring that needs to be corrected.

**Comment by fonnesbeck** [comment]
Williams and Rasmussen uses a -2 multiplier, while GPyTorch uses +2.

**Comment by vr308** [comment]
The power of the exponent should be negative or it would explode (I would think?)

Wikipedia uses -2.

**Comment by fonnesbeck** [comment]
Yes, you're right. It looks like a docstring issue on the GPyTorch side as well.

**Comment by bwengals** [comment]
Made a PR (#3543) which corrects the docstring.

**Comment by fonnesbeck** [comment]
Closed with #3543
